### ISSUE TITLE
Purpose of master and develop branches?

### ISSUE THREAD

**Issue by fgblomqvist** [resonatecoop/stream2own#148]
First time I checked out this project I ended up on master. Then I switched to develop, and then I realized there was also a development branch. The latter is clearly the main one from what I can see. Should the other ones perhaps be removed (for now?)? At the very least, I'm guessing the `develop` branch should go away (should `development` perhaps be renamed to `develop`?).

If the `master` branch is kept, maybe clarify somewhere in the readme what master vs. develop/development means.

Just to reduce confusion for devs looking to contribute :slightly_smiling_face:

**Comment by peterklingelhofer** [comment]
I agree, this clarification would be good to have in the readme, and I'm curious about it myself.

However, I was unable to reproduce how you managed to end up on the `master` branch. When I first cloned this repo, I ended up on `development`. Very strange and curious as to why this happened to you and not me.

I just tried to clone down this repo again, (both resonate's and my own fork, same result both ways) and the same thing happened, I ended up on `development`:
```sh
~ ❯ git clone https://github.com/resonatecoop/stream.git
Cloning into 'stream'...
...
Resolving deltas: 100% (3869/3869), done.
~ ❯ cd stream
~/stream development ❯ 
```

**Comment by fgblomqvist** [comment]
Doing it regularly through command-line I do indeed end up at `development`. I dug into how I ended up on `master` and it turns out that it was due to my git UI. I use [SmartGit](https://www.syntevo.com/smartgit/) (can strongly recommend, free for non-commercial work) and when cloning a repo through it, it defaults to checking out the `master` branch if there is one (seemingly). If there wasn't one, it would perhaps pick `develop`? Or something else.

Either way, I imagine most developers would end up on `development`, but just to make things overly clear, it probably still makes sense to clean house where possible 🙂

**Comment by auggod** [comment]
Both master and develop were stale for a long time. I have removed them. Develop was useful at a time when we were running two versions of the player at the same time.

**Comment by fgblomqvist** [comment]
Makes sense, good cleanup. Should `development` perhaps also be renamed to `develop` to match the other resonate repos and general got conventions?

**Comment by auggod** [comment]
I thought we would do the opposite, renaming all other repos develop branches to development instead. Anyway, not critical right now.
